### ISSUE TITLE
1.19.2 Launch crash - Mixin conflict

### ISSUE THREAD

**Issue by ZsoltMolnarrr** [jellysquid3/Lithium#458]
### Version Information

lithium-fabric-mc1.19.2-0.11.0.jar

### Expected Behavior

Not crash on launch.

### Actual Behavior

Mixin conflict crash on launch.

### Reproduction Steps

Not sure which mod causes the problem. Log attached.
Feel free to close in case this issue doesn't belong here.

### Other Information

[latest.log](https://github.com/CaffeineMC/lithium-fabric/files/10978899/latest.log)

Downgrading to Lithium 0.10.4 makes the launch crash not to occur.

**Comment by adolf1889** [comment]
Same problem here
```
[20:13:29] [main/ERROR]: Critical injection failure: LVT in net/minecraft/class_3924::method_31666(Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_3924;)V has incompatible changes at opcode 109 in callback lithium.mixins.json:world.block_entity_ticking.sleeping.campfire.CampfireBlockEntityMixin from mod lithium->@Inject::trySleepLit(Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_3924;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Z)V.
 Expected: [Z]
    Found: [Lme/isaiah/common/R117/ICampfireBlockEntity;]
Available: [Lme/isaiah/common/R117/ICampfireBlockEntity;, I, Lnet/minecraft/class_1799;, Lnet/minecraft/class_1277;, Lnet/minecraft/class_1799;, Lnet/minecraft/class_2338;, Lme/isaiah/common/event/entity/CampfireBlockEntityCookEvent;]
[20:13:29] [main/ERROR]: Minecraft has crashed!
```

**Comment by OxiiGiin1** [comment]
Also get this issue on my server.

[https://pastebin.com/ntuV04Zg](url)

**Comment by CantWeAllDisagree** [comment]
Same Issue crashing client and server. Minecraft 1.19.2

Server Logs:
https://pastebin.com/BUWxu2PU

**Comment by bananaOHHHOOOOHAHAHAHAAAAHA** [comment]
i also get this for my modpack. samething involving campfires

**Comment by bananaOHHHOOOOHAHAHAHAAAAHA** [comment]
also version 0.10.4 doesnt fix it for me

**Comment by 2No2Name** [comment]
Fixed in 0.11.1
